local ReplicatedStorage = game:GetService("ReplicatedStorage")

local UILib = require(ReplicatedStorage.UILib)

local window = UILib:CreateWindow({
	Title = "PixlLib",
})

window:createTab("Home", 15637356511)
window:createTab("Settings", 15637356511)
window:createTab("About", 15637356511)
window:CreateTab("Info", 16089599792)
window:CreateTab("Eggs", 15637367451)
window:CreateTab("Farming", 16093555964)
window:CreateTab("Settings", 87867412432006)
window:CreateTab("Player", 15637355840)
window:CreateTab("Upgrades", 16089599792)
window:CreateTab("Fishing", 16093555964)
window:CreateTab("Mailbox", 15637375511)
window:CreateTab("Quests", 15637359494)
window:CreateTab("Automatic", 15637369009)
window:CreateTab("Webhook", 122219359034661)
window:CreateTab("Statistics", 94231924171549)
window:CreateTab("Optimisation", 94231924171549)
window:CreateTab("Shop", 16093555964)
